rosidl_generator_py/cmake/rosidl_generator_py_generate_interfaces.cmake

```diff
@@ -209,6 +219,11 @@ foreach(_typesupport_impl ${_typesupport_impls})
     ${rosidl_generate_interfaces_TARGET}${_target_suffix}
     ${rosidl_generate_interfaces_TARGET}__rosidl_typesupport_c
   )
+  add_dependencies(
+    ${_target_name_lib}
+    ${rosidl_generate_interfaces_TARGET}${_target_suffix}
+    ${rosidl_generate_interfaces_TARGET}__rosidl_typesupport_c
```

Comment:
Along the same lines, if we don't need to depend on specific typesupport, we may also be able to generate a single `_python_` library instead of 1 per typesupport

Reply:
Ok I gave that a shot at https://github.com/ros2/rosidl_python/commit/7fbb83234e56a1660ac62fef6dd1ad556b847795 and it seems to work fine with a single python library per package